protocols/v2/noise-sv2/README.md

```diff
@@ -0,0 +1,36 @@
+
+<div align="center">
+<h1>Noise-SV2</h1>
+<p>
+    A noise protocol implemenation for Stratum V2
+</p>
+<p>
+    <a href="https://github.com/stratum-mining/stratum/blob/main/LICENSE.md"><img alt="CC0 1.0 Universal Licensed" src="https://img.shields.io/badge/license-CC0--1.0-blue.svg"/></a>
+    <a href="https://app.codecov.io/gh/stratum-mining/stratum/tree/main/protocols%2Fv2%2Fnoise-sv2">
+    <img src="https://codecov.io/gh/stratum-mining/stratum/branch/main/graph/badge.svg" alt="codecov">
+</a>
+    <a href="https://blog.rust-lang.org/2023/12/28/Rust-1.75.0.html"><img alt="Rustc Version 1.75.0+" src="https://img.shields.io/badge/rustc-1.75.0%2B-lightgrey.svg"/></a>
+    <a href="https://crates.io/crates/noise_sv2"><img alt="crate.io" src="https://img.shields.io/crates/v/noise_sv2.svg"/></a>
+    <a href="https://docs.rs/noise_sv2"><img alt="docs" src="https://docs.rs/noise_sv2/badge.svg"/></a>
+  </p>
+</div>
+
+
+This crate provides an implementation of the [Noise protocol](https://github.com/stratum-mining/sv2-spec/blob/main/04-Protocol-Security.md#4-protocol-security) as outlined in the Stratum V2 specification. The `noise_sv2` crate ensures secure communication channels between downstream and upstream components of the Stratum V2 mining protocol by performing encryption, decryption, and authentication via Noise handshakes and cipher operations.
+
+## Key Features
```

Comment:
again, I think we should avoid the word "features" as much as possible because this can easily be confused with `--features` compilation flag